inspirehep/modules/literaturesuggest/tasks.py

```diff
@@ -249,22 +83,20 @@ def curation_ticket_context(user, obj):
         "doi:{0}".format(doi)
         for doi in get_value(obj.data, 'dois.value') or []
     ]
-    link_to_pdf = obj.extra_data.get('formdata', {}).get('url')
+    link_to_pdf = obj.extra_data.get('submission_pdf')
 
     subject = ' '.join(filter(
         lambda x: x is not None,
         arxiv_ids + dois + report_numbers + ['(#{0})'.format(recid)]
     ))
 
-    references = obj.extra_data.get('formdata').get('references')
-    user_comment = obj.extra_data.get('formdata', {}).get('extra_comments', '')
+    user_comment = _get_user_comment(obj)
 
     return dict(
         recid=recid,
         record_url=record_url,
         link_to_pdf=link_to_pdf,
         email=user.email,
-        references=references,
```

Comment:
Why remove this field?

Reply:
See the old comment https://github.com/inspirehep/inspire-next/pull/3081#issuecomment-353637476

The reason behind it is to be able to move `formdata_to_model` to a non-task function, and thus, simplify it's usage so we don't need any dummy mocks in the tests for it anymore.

If we can avoid having it as a task, we should :), even if we need it as a task, I'd leave the task as a very simple wrapper on top of the actual function that does the stuff.

Reply:
At some point we might want to create a 'builder' class, or just a class for the formdata, same as we do for the record, that would help using the formdata, and testing it too. So you can do stuff like:
```python
my_formdata = Formdata(field1='bla', field2='bla')
function_that_uses_formdata(Formdata)

# or
my_formdata = Formdata(invalid_field='dummy value')
# raises TypeError exception with 'got unexpected keyword argument invalid_field'
```

That way, every time we modify it we will notice if anyone that uses it is expecting it to have certain fields.